bigquery/google/cloud/bigquery/job.py

```diff
@@ -640,6 +641,19 @@ def schema(self, value):
             raise ValueError('Schema items must be fields')
         self._schema = tuple(value)
 
+    @property
+    def kms_key_name(self):
+        """See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.destinationEncryptionConfiguration.kmsKeyName
+        """
+        return self._kms_key_name
+
+    @kms_key_name.setter
+    def kms_key_name(self, value):
+        if not isinstance(value, six.string_types) and value is not None:
+            raise ValueError("Pass a string, or None")
```

Comment:
Error message should say what property this is.

You could potentially drop the type check entirely, as in general we prefer to let the server do request validation.

Reply:
This is consistent with the rest of the file